dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp

```diff
@@ -1,10 +1,10 @@
 <%@ page import="com.liferay.portal.language.LanguageUtil" %>

```

Comment:
Where is the code to make the related content dialog bigger?

Reply:
dojo take as reference ` .related-content-form ` to set the dimensions